tests/unit/metrics-adapters/mixpanel-test.js

```diff
@@ -49,6 +49,17 @@ test('#trackEvent calls `mixpanel.track` with the right arguments', function(ass
   assert.ok(stub.secondCall.calledWith('Ate a cookie'), 'it sends the correct arguments');
 });
 
+test('#trackPage calls `mixpanel.track` with the right arguments', function(assert) {
+  var adapter = this.subject({ config });
```

Comment:
That seems to be very controversial question of late ;)

I was following the convention in the rest of the test obviously ;)

I'll change them all.